### PR TITLE
roll back readme

### PR DESCRIPTION
**Pull request by yiheng**
roll back readme to an earlier version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,33 +27,12 @@ Step-by-step Deep Leaning Tutorials on Apache Spark using [BigDL](https://github
...
-
-   Set SPARK_HOME variable
-    ```
-    export SPARK_HOME="Downloaded Spark located directory"
-    ```
-   Use prepared *start_notebook.sh* to start jupyter server
-    ```
-    bash start_notebook.sh
-    ```
+* Run ```pip install BigDL==0.3.0```
```
Comment (dding3) [inline README.md]: why 0.3.0 version in master branch?

Comment (yiheng) [reply README.md]: I think we forget to update it, has fixed

Comment (dding3) [reply README.md]: I see. I noticed in both branch-0.3 and branch-0.4, the version is 0.3.0. Please update branch-0.4 as well?

**Comment by dding3** [comment]
Another thing I noticed in rdd.ipynb under spark_basis dir, it uses a built-in sc which doesnot exist if we don't use start_notebook.sh to start jupyter. I think we need create it as "sc=SparkContext.getOrCreate(conf=create_spark_conf().setMaster("local[4]").set("spark.driver.memory","2g"))" like what we did in neural network notebooks.

**Comment by yiheng** [comment]
@megaSpoon why we change the ReadMe in earlier change

**Pull request merged**